msal/src/main/java/com/microsoft/identity/client/UiBehavior.java

```diff
@@ -36,11 +36,27 @@
 
     /**
      * acquireToken will send prompt=login to the authorize endpoint.  The user will always be prompted for credentials by the service.
+     *
+     * toString override is to enable the correct protocol value of login to be returned instead of "force_login".
      */
     FORCE_LOGIN,
 
     /**
      * acquireToken will send prompt=consent to the authorize endpoint.  The user will be prompted to consent even if consent was granted before.
      */
-    CONSENT
+    CONSENT;
+
+    @Override
+    public String toString(){
+        switch(this){
+            case SELECT_ACCOUNT:
+                return SELECT_ACCOUNT.name().toLowerCase();
+            case FORCE_LOGIN:
+                return "login";
+            case CONSENT:
+                return CONSENT.name().toLowerCase();
+            default:
+                throw new IllegalArgumentException();
```

Comment:
`IllegalStateException` may be a closer representation of what went wrong here, but the outcome is the same (aka `RuntimeException`)